package.json

```diff
@@ -79,6 +79,21 @@
             "type": "boolean",
             "description": "Enable the new query panel in dbt.",
             "default": true
+          },
```

Comment:
Could be better if all these settings are wrapped under `lineage` as single config? OR add prefix `lineage` for each field so it will be easy to relate